test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -921,6 +921,36 @@ public void WordEmbeddingsTest()
             Done();
         }
 
+        [Fact]
+        public void TokenizingByCharactersOnnxConversionTest()
+        {
+            var mlContext = new MLContext(seed: 1);
+            var dataPath = GetDataPath("wikipedia-detox-250-line-test.tsv");
+            var dataView = ML.Data.LoadFromTextFile(dataPath, new[] {
```

Comment:
Are there no issues with utf-8 encoding here?

Reply:
yes, the issues happen with the wikipedia-detox-250-line-data.tsv file

Reply:
Does the test run successfully? If not is there a different file you can use? Or just used fixed inline data.

---
In reply to: [376146626](https://github.com/dotnet/machinelearning/pull/4805#discussion_r376146626) [](ancestors = 376146626)

Reply:
Yes, I'm using wikipedia-detox-250-line-**test**.tsv instead

Reply:
Ah..ok. Sounds good.